test/elpa.el

```diff
@@ -0,0 +1,5 @@
+(setq package-user-dir
+      (expand-file-name (format ".cask/%s/elpa" emacs-version)))
```

Comment:
Does Cask still have problem being used from `ansi-term`? That was another issue I had earlier.

In any case, I think it's more flexible to tweak a few lines of an Elisp build target rather than relying on a separate Python script and passing data from Makefile through shell interface.

For instance, see `(setq byte-compile--use-old-handlers nil)`. Looks better in an Elisp file rather than being passed through `--eval` in a Makefile.